scripts/wikitables/generate_data_from_erm_model.py

```diff
@@ -0,0 +1,67 @@
+#! /usr/bin/env python
+
+# pylint: disable=invalid-name,wrong-import-position,protected-access
+import sys
+import os
+import gzip
+import argparse
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir)))))
+
+from allennlp.data.dataset_readers import WikiTablesDatasetReader
+from allennlp.models.archival import load_archive
+from allennlp.training.metrics import WikiTablesAccuracy
+
+
+def make_data(input_examples_file: str,
+              tables_directory: str,
+              archived_model_file: str,
+              output_dir: str,
+              num_logical_forms: int) -> None:
+    reader = WikiTablesDatasetReader(tables_directory=tables_directory,
+                                     keep_if_no_dpd=True,
+                                     output_agendas=True)
+    dataset = reader.read(input_examples_file)
+    input_lines = open(input_examples_file).readlines()
```

Comment:
Using a `with` block to read the file would be better here.

Reply:
Done.